### PR TITLE
Support for Arduino Core 2.6.1....

### PR DESCRIPTION
**Pull request by Jason2866**
and deleted all old outdated cores

## Checklist:
  - [x] The pull request is done against the latest dev branch
  - [x] Only relevant files were touched
  - [x] Only one feature/fix was added per PR.
  - [x] The code change is tested and works on core 2.6 and core 2.6.1
  - [x] The code change pass travis tests. **Your PR cannot be merged unless tests pass**
  - [x] I accept the [CLA](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md#contributor-license-agreement-cla).

### CODE REVIEW

**Comment by andrethomas** [comment]
Too quick... fix the tasmota part should not be included :)

**Comment by Jason2866** [comment]
Thx!

File: platformio.ini [hunk 1]
```diff
@@ -185,18 +161,25 @@ build_flags               = ${esp82xx_defaults.build_flags}
...
-; *** Arduino Esp8266 core pre 2.6.x for Tasmota (recommended version, no known issues)
-platform                  = https://github.com/Jason2866/platform-espressif8266.git#Tasmota
+[core_2_6_1]
+; *** Esp8266 core for Arduino version 2.6.1 (for Windows, most Linux variants and Mac)
+; *** custom setup until the core 2.6.0 version is official released from PlatformIO crew
+platform                  = https://github.com/Jason2866/platform-espressif8266.git#core_2_6_1
 build_flags               = ${esp82xx_defaults.build_flags}
-                            -Wl,-Tesp8266.flash.1m.ld
+                            -Wl,-Teagle.flash.1m.ld
                             -O2
```
Comment (s-hadinger) [inline platformio.ini]: It is unrelated to this PR but the -O2 does not have any effect in platform.io. It is overridden by a -Os by platformio, which is good.

Comment (andrethomas) [reply platformio.ini]: I've been wondering what the -O2 is doing there...

**Pull request merged**